README.md

```diff
@@ -168,3 +162,7 @@ For contacting the Zonemaster project, please send a mail to
 [zonemaster-gui]: https://github.com/dotse/zonemaster-gui
 [ldns]: https://www.nlnetlabs.nl/projects/ldns/
 [CPAN]: http://search.cpan.org/search?query=Zonemaster&mode=dist
+
+[Zonemaster-LDNS issue 10]: https://github.com/dotse/zonemaster-ldns/issues/10
+[Zonemaster-LDNS issue 30]: https://github.com/dotse/zonemaster-ldns/issues/30. 
+
```

Comment:
Why do we need to add the LDNS issue here, keeping in mind that Debian 8 is removed from the supporting OS

Reply:
I have created a reference that I use on line 55.